docs/modules/ROOT/pages/accounts.adoc

```diff
@@ -78,7 +78,12 @@ In this section we will describe the methods that the protocol uses for abstract
 are required for enabling accounts to be used for executing transactions. The rest are optional:
 
 1. `\\__validate__` verifies the validity of the transaction to be executed. This is usually used to validate signatures,
+<<<<<<< HEAD
+but following the account abstraction design, the entrypoint implementation can be customized to feature any
+validation mechanism.
+=======
 but the entrypoint implementation can be customized to feature any validation mechanism https://docs.starknet.io/documentation/architecture_and_concepts/Accounts/validate_and_execute/#validate_limitations[with some limitations].
+>>>>>>> cairo-2
```

Comment:
Unresolved merge conflict.

Reply:
reminded me of
https://twitter.com/VictorBuldakov/status/1666527326815899663